### PR TITLE
Walk Score and Description

### PR DESCRIPTION
**Pull request by denced**
[Walk Score and Description: Trello Card](https://trello.com/c/qimOfH8O)

Location:

-Map.storyboard
-LocationDView.swift .. deleted
-DetailViewController.swift
-WalkScore.swift
-NetworkController.swift

Commit Details:

Purged mock api request and created a walk score api request
Displayed results on detail view

### CODE REVIEW

**Comment by denced** [comment]
[Walk Score request](https://trello.com/c/qimOfH8O/59-walk-score-request)

File: labs-ios-starter/Map/MapViewController.swift [hunk 1]
```diff
@@ -107,40 +98,17 @@ class MapViewController: UIViewController, MKMapViewDelegate {
...
-        guard !isCurrentlyFetchingWS else {
-            //then we want to remember to refresh once request comes back
-            shouldRequestWSAgain = true
-            return
-        }
-        
-        isCurrentlyFetchingWS = true
-        let visibleRegion = mapView.visibleMapRect
+ 
+        WalkingScoreNetworkController.fetchWalkScore(lat: lat, lon: lon, completion: { (walkScore, error) in
```
Comment (alext8900) [inline labs-ios-starter/Map/MapViewController.swift:119]: I'm glad passing the lat and Lon as parameters actually worked. Nice!

**Review by alext8900 — APPROVED** [review]
Good job denis.

**Review by bronsonmullens — APPROVED** [review]
Great job getting that api connected.

**Pull request merged**